src-vue/src/components/PlayButton.vue

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { NorthstarState } from '../utils/NorthstarState';
+import { ReleaseCanal } from '../utils/ReleaseCanal';
+
+export default defineComponent({
+    name: 'PlayButton',
+    computed: {
+        playButtonLabel(): string {
+            if (this.$store.state.northstar_is_running) {
+                return "Game is running";
+            }
+
+            switch(this.$store.state.northstar_state) {
+                case NorthstarState.INSTALL:
+                    return "Install";
+                case NorthstarState.INSTALLING:
+                    return "Installing..."
+                case NorthstarState.MUST_UPDATE:
+                    return "Update";
+                case NorthstarState.UPDATING:
+                    return "Updating...";
+                case NorthstarState.READY_TO_PLAY:
+                    return "Launch game";
+
+                default:
+                    return "";
+            }
+        },
+        northstarIsRunning(): boolean {
+            return this.$store.state.northstar_is_running;
+        },
+        options(): {key: string, value: string}[] {
+            return Object.keys(ReleaseCanal).map(function (v) {
+                return {
+                    key: v,
+                    // @ts-ignore
```

Comment:
Another `@ts-ignore` :eyes:

Reply:
Fixed in 02127ed8793e5344c28fa9c7c2c16c7a0b3f3ae9